cookie-banner-rules-list.json

```
@@ -6370,10 +6370,15 @@
     {
       "id": "98D89E26-F4B6-4C2D-BABF-4295B922E433",
       "domains": [
+        "auto5.be",
         "bouyguestelecom.fr",
         "enedis.fr",
         "fortuneo.fr",
         "lcl.fr",
+        "norauto.es",
```

While you are at it, could you please add the following domains to the list as well:
- norauto-franchise.com
- norauto-pro.com

```suggestion
        "norauto-franchise.com",
        "norauto-pro.com",
        "norauto.es",
```